barretenberg/cpp/src/barretenberg/stdlib/recursion/honk/verifier/protogalaxy_recursive_verifier.cpp

```diff
@@ -152,8 +152,24 @@ std::shared_ptr<typename VerifierInstances::Instance> ProtoGalaxyRecursiveVerifi
     auto lagranges = std::vector<FF>{ FF(1) - combiner_challenge, combiner_challenge };
 
     auto next_accumulator = std::make_shared<Instance>(builder);
-    next_accumulator->verification_key = accumulator->verification_key;
-    next_accumulator->verification_key->log_circuit_size = accumulator->verification_key->log_circuit_size;
+    next_accumulator->verification_key = std::make_shared<VerificationKey>(
```

Comment:
not the best pattern, could probably clean up as next_accumulator->verification_key = accumulator->verification_key in a clarifying state PR